obspy/io/seiscomp/inventory.py

```diff
@@ -494,12 +553,13 @@ def _read_instrument_sensitivity(sen_element, cha_element, _ns):
     return sensitivity
 
 
-def _read_response(root, sen_element, resp_element, cha_element,
-                   data_log_element, _ns, samp_rate, fir, analogue):
+def _read_response(instrumentation_register, sen_element, resp_element,
```

Comment:
It doesn't look like you need all of `instrumentation_register` here, so I would pass `instrumentation_register['responses']` to this function. Then below you get shorter lines and less directory indexing.

Reply:
Changed as suggested.